src/lib/config.ts

```diff
@@ -108,4 +110,14 @@ export interface Config {
 	 * Port to run the local server on.
 	 */
 	port?: number;
+
+	/**
+	 * The text to display on top of the table of content. Default to 'Table of content'
+	 */
+	toc_headings?: string;
```

Comment:
```suggestion
	toc_heading: string;
```

This option has a default value so it doesn't need to be possibly undefined.